packages/web/src/pages/earn/pool/[pool-path]/add.tsx

```diff
@@ -25,15 +24,15 @@ export default function EarnAdd() {
       { title: "Earn", path: "/earn" },
       {
         title:
-          breakpoint === DEVICE_TYPE.WEB
+          width >= 1250
```

Comment:
Declare a variable corresponding to 1250.

Reply:
done